init-plugin.sh

```diff
@@ -65,11 +66,12 @@ cd tests
 git mv test-foo-bar.php "test-$slug.php"
 cd ..
 
-perl -p -i'' -e "s/Foo Bar/$name/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/foo-bar/$slug/g" $( find */ -type f  ) *.*
-perl -p -i'' -e "s/foo_bar/$prefix/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/FooBar/$namespace/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/Foo_Bar/$class/g" $( find */ -type f ) *.*
+git grep -lz "Foo Bar" | xargs -0 sed -i '' -e "s/Foo Bar/$name/g"
+git grep -lz "foo-bar" | xargs -0 sed -i '' -e "s/foo-bar/$slug/g"
+git grep -lz "foo_bar" | xargs -0 sed -i '' -e "s/foo_bar/$prefix/g"
+git grep -lz "FooBar" | xargs -0 sed -i '' -e "s/FooBar/$namespace/g"
+git grep -lz "Foo_Bar" | xargs -0 sed -i '' -e "s/Foo_Bar/$class/g"
```

Comment:
@westonruter and @lukecarbis please note that these sed commands with `-i ''` do not work on Linux.

Reply:
Oh yeah! I remember now that's why I used `perl` to begin with.